lib/fabrik/DirectorInstance.js

```diff
@@ -50,37 +67,62 @@ class DirectorInstance extends BaseInstance {
         this.networkSegmentIndex = networkSegmentIndex;
       })
       .tap(() => {
+        //TODO: This still has coupling with space guid as it calls ensureTenantId. 
+        //As part of SF 2.0, we can go to ETCD and fetch the tenant_id which will 
+        //ensure we dont call CF again. For now, we add a check for the feature validation.
```

Comment:
Not sure if this commend is valid. Code is moved to platformManager. So is this comment still valid?

Reply:
Removed